src/character.cpp

```diff
@@ -5776,7 +5790,10 @@ void Character::update_stamina( int turns )
 {
     static const std::string player_base_stamina_regen_rate( "PLAYER_BASE_STAMINA_REGEN_RATE" );
     static const std::string stamina_regen_modifier( "stamina_regen_modifier" );
+    static const std::string stamina_cardiofit_regen_modifier( "PLAYER_CARDIOFIT_STAMINA_MOD" );
     const float base_regen_rate = get_option<float>( player_base_stamina_regen_rate );
+    const float effective_regen_rate = base_regen_rate + get_bmr() * get_option<float>
```

Comment:
Should this be using get_bmr() or (possibly modified) base_bmr()? The former includes current activity levels.

Reply:
```suggestion
    const float effective_regen_rate = base_regen_rate + base_bmr() * get_option<float>
```